### ISSUE TITLE
[FEAT] - accents support for non-english keyboards

### ISSUE THREAD

**Issue by darkvertex** [furcan/KioskBoard#91]
### Is your feature request related to a problem? Please describe.

The additional language key profiles are a welcome addition but unfortunately it does not contain anything to deal with accents.

### Describe the solution you'd like

I can't speak for languages I am not a master of, but as a Spaniard I expect the Spanish profile to at minimum offer our two accent-insertion keys, normally near the "p" and the "ñ":

![image](https://github.com/furcan/KioskBoard/assets/941331/ae1eafa1-d429-4565-86c1-a11001e1f5dd)

One puts an accent leaning right, the other leaning left. The way it works in a Spanish keyboard is you press the particular accent key, then follow with a vowel key, it types an accented vowel.

In other words:
´ + a = á (example: "Mamá" / "Mother")
` + a = à (example: "hablò" / "[he] spoke")

I imagine the accenting behaviour acts similarly in other languages like French.

While we're talking about this, symbols should be part of your localized key profiles too, not just the letters. For example, Spanish has upside down versions of the question (¿?) and exclamation (¡!) marks, which Kiosk Keyboard cannot currently insert.
